packages/player/src/styles/scss/components/control-bar.scss

```diff
@@ -1,10 +1,7 @@
 .cueplayer-react .cueplayer-react-control-bar {
   display: flex;
   width: 100%;
-  position: absolute;
-  bottom: 0;
-  left: 0;
-  right: 0;
+  // removes absolute positioning which should be optional for "overlay" style
```

Comment:
this lets it stack, but I feel like the absolute positioning should be optional to the control bar so it COULD overlay. Generally we will use it stacked below the video though